contract/src/main/java/io/pravega/schemaregistry/contract/generated/rest/model/CodecTypesList.java

```diff
@@ -29,16 +30,16 @@
 
 public class CodecTypesList   {
   @JsonProperty("codecTypes")
-  private List<String> codecTypes = null;
+  private List<CodecType> codecTypes = null;
 
-  public CodecTypesList codecTypes(List<String> codecTypes) {
+  public CodecTypesList codecTypes(List<CodecType> codecTypes) {
```

Comment:
`CodeTypes` would be a better name for this class, although this class wasn't added as part of this PR.